scripts/cci-workflow-results.ts

```diff
@@ -0,0 +1,95 @@
+import { join } from "path";
+import { getOldJobNameWithoutSuffixes, getTestFiles, getTimingsFromJobsData, REPO_ROOT, saveTestTimings, saveWorkflowResults, saveWorkflowResultsHTML } from "./cci-utils";
+import axios from 'axios';
+
```

Comment:
todo: refactor and move CCI api calls to a different file - those should be exported/common methods